components/feature/intent/src/main/java/mozilla/components/feature/intent/processing/TabIntentProcessor.kt

```diff
@@ -73,7 +74,7 @@ class TabIntentProcessor(
         } else {
             val url = WebURLFinder(extraText).bestWebURL()
             if (url != null) {
-                val session = createSession(url, private = isPrivate, source = Source.ACTION_SEND)
```

Comment:
Why do you drop all the private flags here..?

Reply:
They all pass in the same value (`isPrivate`).

Reply:
Oh, I somehow missed that this is calling `createSession` and is not the `Session()` constructor :)